src/um/audioclient.rs

```diff
@@ -51,21 +51,68 @@ pub const AUDCLNT_E_RAW_MODE_UNSUPPORTED: HRESULT = AUDCLNT_ERR!(0x027);
 pub const AUDCLNT_S_BUFFER_EMPTY: SCODE = AUDCLNT_SUCCESS!(0x001);
 pub const AUDCLNT_S_THREAD_ALREADY_REGISTERED: SCODE = AUDCLNT_SUCCESS!(0x002);
 pub const AUDCLNT_S_POSITION_STALLED: SCODE = AUDCLNT_SUCCESS!(0x003);
+pub const AUDCLNT_S_NO_SINGLE_PROCESS: SCODE = AUDCLNT_SUCCESS!(0x00d);
```

Comment:
Where is this definition from? I can't find it in the Windows 10.0.19041 SDK

Reply:
This was in @trmcnvn commit. @trmcnvn I also can't seem to find this can you remember where you got this from?

Reply:
See my comment on the old PR here: https://github.com/retep998/winapi-rs/pull/796#discussion_r424731119

I needed this constant in a personal project so I included it in my fork/branch. I wasn't sure if it should be in winapi-rs or user code as it isn't part of the SDK defs.

Reply:
Since it isn't in the Windows SDK headers, it should be defined in user code and not winapi.

Reply:
Thanks for that, i'll remove this line then.